src/main/java/frc/robot/subsystems/LimelightNavigation.java

```diff
@@ -46,7 +46,7 @@ public class LimelightNavigation extends SubsystemBase {
   // MecanumDriveOdometry m_DriveOdometry;
   static MecanumDrivePoseEstimator m_DrivePoseEstimator;
 
-  public static Pigeon2 m_pigeon2;
+  private Pigeon2 m_pigeon2;
```

Comment:
I would make this `private final`. The `final` keyword means the variable won't be reassigned after the first assignment, which is going to be true of things like subsystems and hardware components.

Reply:
Done.